keras_core/ops/math.py

```diff
@@ -1,5 +1,7 @@
 """Commonly used math operations not included in NumPy."""
 
+import scipy.signal
```

Comment:
We should not depend on scipy if at all possible.

Reply:
I have refactored the implementation of `get_window`

The biggest change involves transferring the logic of `stft` from `keras_core.ops.math` to `keras_core.backend.*.math`

- tf: `tf.signal.hann_window` and `tf.signal.hamming_window`
- torch: `torch.hann_window` and `torch.hamming_window`
- jax & numpy: `scipy.signal.get_window`